src/App.tsx

```diff
@@ -25,10 +25,12 @@ export class App extends React.PureComponent<{}, AppState> {
 
 	state: AppState = {
 		dayNightStatus: false,
-		currentIPInput: '192.168.57.30',
+		NewIPInput: '',
 		currentButterClients: [],
-		show: false,
+		showInst: false,
+		showNewIP: false,
 		labCurrentIPs: ['192.168.57.30', '192.168.57.32', '192.168.57.34', '192.168.56.188', '192.168.56.193', '192.168.56.206'],
+		IPdeleteState: [false, false, false, false, false, false],
```

Comment:
what happens to this list after you delete and then add new ip?

Reply:
make sure that you don't mix states with different ip's after deleting and adding some

Reply:
you can see in the code, where you add and remove IPs, it updates the new array as well, and I tested it and it works.
It checks the index of the IP in the IPs array, and when removing it, it removes the corresponding index in the states array.